src/scripts/index.js

```diff
@@ -0,0 +1,56 @@
+let books = [];
+const addButton = document.querySelector('.add-btn');
+const bookList = document.querySelector('.books-ul');
+const storage = window.localStorage;
+
+function Book(name, title) {
+  this.name = name;
+  this.title = title;
+}
+
+function addBook() {
+  const bookName = document.querySelector('.bookname').value;
+  const bookTitle = document.querySelector('.author').value;
+
+  const book = new Book(bookName, bookTitle);
+  books.push(book);
```

Comment:
- [ ] _[OPTIONAL]_ Using array methods that doesn't mutate the array, e.g. `concat`, `filter`, `slice` instead of `push`, `pop`, `splice` and so on. :ok_hand: